libobs/util/platform-windows.c

```diff
@@ -1294,6 +1294,14 @@ uint64_t os_get_sys_free_size(void)
 	return msex.ullAvailPhys;
 }
 
+uint64_t os_get_sys_total_size(void)
+{
+	MEMORYSTATUSEX msex = {sizeof(MEMORYSTATUSEX)};
+	if (!os_get_sys_memory_usage_internal(&msex))
+		return 0;
+	return msex.ullTotalPhys;
```

Comment:
Prefer a blank line between the `if` body and the following `return` statement, or use braces for the `if` and the `return` can go on the line immediately after the closing brace.